src/dcp/dcp.c

```diff
@@ -85,6 +173,22 @@ int main(int argc, char** argv)
     /* assume we'll exit with success */
     int rc = 0;
 
+    /* DAOS vars */ 
+#ifdef DAOS_SUPPORT
```

Comment:
Since these are variables that will be set by the option processing, let's move those down to around to where the other variables like input are.

Also, let's initialize these so they are well defined with some value in the case the user does not set them through command line options.

Reply:
I didn't check, but let's also be sure to order the options in the same way in the help message, their declaration in getopt, and the code that processes them.

Reply:
@adammoody I verified they are in that order. It goes src pool, dest pool, src cont, dest cont, svcl, then the daos prefix.